container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -788,6 +789,7 @@ parameter_types! {
     pub BlockGasLimit: U256 = U256::from(BLOCK_GAS_LIMIT);
     pub PrecompilesValue: TemplatePrecompiles<Runtime> = TemplatePrecompiles::<_>::new();
     pub WeightPerGas: Weight = Weight::from_parts(weight_per_gas(BLOCK_GAS_LIMIT, NORMAL_DISPATCH_RATIO, WEIGHT_MILLISECS_PER_BLOCK), 0);
+    pub SuicideQuickClearLimit: u32 = 0;
```

Comment:
check what this implies!

Reply:
it is to delete the storage associated with the contracts up to the limit: https://github.com/paritytech/frontier/pull/1221.

Setting it to 0 maintains the same behavior as before, but for future releases, maybe we should think about setting this value